packages/appstore/src/components/fiat-transaction-list/fiat-transaction-list.scss

```diff
@@ -0,0 +1,49 @@
+.fiat-transaction-list {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+    margin: 0 auto;
+    width: 100vw;
+    max-width: 800px;
+
+    &__day {
+        display: flex;
+        flex-direction: column;
+
+        & > span {
+            padding: 8px 16px;
+        }
+
+        & > div {
+            box-shadow: inset 0 1px 0 #f2f3f4;
+        }
+    }
+
+    &__item {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+        align-items: center;
+        height: 72px;
```

Comment:
Do we need to define the exact height of the item? I think it can be based on the content.

Reply:
Done